test/hotspot/jtreg/runtime/ClassUnload/UnloadInterfaceTest.java

```diff
@@ -86,8 +88,11 @@ private static void run() throws Exception {
         ClassUnloadCommon.failIf(!wb.isClassAlive(interfaceName), "should be live here");
 
         cl = null; c = null; o = null;
-        ClassUnloadCommon.triggerUnloading();
-        ClassUnloadCommon.failIf(wb.isClassAlive(className), "should have been unloaded");
+
+        // Now unload className. This calls triggerUnloading but we only pass the class we expect to be unloaded
+        // otherwise the test will take too long.
```

Comment:
Don't understand why this one needs a comment and none of the others do. ?? Also why the "otherwise" part?

Reply:
I was trying to explain why interfaceName is not in the List.  I could have added it and tested that it's not unloaded but then the test would take too long.  It may be more correct to add it and make triggerUnloading try to unload it and fail to prove it cannot be unloaded.  But like I said in the comment, it would make the test take too long.

Reply:
Seems pretty evident from the `failIf` calls what is expected. The comment is just confusing to me.

Reply:
I guess it's confusing but I'm trying to say why not this:
```
ClassUnloadCommmon.triggerUnloading(List.of(className, interfaceName));
ClassUnloadCommon.failIf(wb.isClassAlive(className), "should be unloaded");
ClassUnloadCommon.failIf(!wb.isClassAlive(interfaceName), "should be live here");
```
Which would be a better test since interfaceName should not be unloaded because it's referenced and not because we didn't try hard enough in the loop to wait for other things like the compiler to stop referencing it.  But I didn't want to do that because it would make the test run longer in order to wait for it not to unload.